var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -49,6 +58,11 @@ class Relion(CMakePackage):
     depends_on('fftw+float+double')
     depends_on('fltk', when='+gui')
     depends_on('cuda@8.0:8.99', when='+cuda')
+    # compiles with gcc > 5 under cuda8
+    conflicts('%gcc@5.0.0:9.0.0', when='+cuda')
```

Comment:
This conflict confuses me a bit given the associated comment: the comment implies to me that gcc > 5 is desirable to build this package. The `conflict` creates an error whenever `5 <= gcc version <= 9`. Did you mean the comment to read:

> compiles with gcc < 5 for cuda 8

?

If so, IMO the `conflicts` constraint could be `conflicts('%gcc@5:', when='+cuda')` unless you anticipate some gcc version > `9.0.0` to resolve this issue.

Reply:
Yes, correct, when using cuda8 gcc less than 5 must be used.